shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/AboutConfScreen.kt

```diff
@@ -114,11 +114,11 @@ fun AboutConfSchedule() {
 private fun AboutConfDescription() {
     Column(Modifier.background(MaterialTheme.colors.whiteGrey)) {
         val text =
-            """KotlinConfKotlinConf is the official annual conference devoted to the Kotlin programming language. Organized by JetBrains, it is a place for the community to gather and discuss all things Kotlin."""
+            """KotlinConf is the official annual conference devoted to the Kotlin programming language. Organized by JetBrains, it is a place for the community to gather and discuss all things Kotlin."""
```

Comment:
Don't you want to extract it to the strings resources?

Reply:
thanks, done